application/start-cron.sh

```diff
@@ -1,22 +1,7 @@
 #!/usr/bin/env bash
 
-# Dump env to a file
-touch /etc/cron.d/idsync
-env | while read line ; do
-   echo "$line" >> /etc/cron.d/idsync
-done
-
-# Add env vars to idsync-cron to make available to scripts
-cat /etc/cron.d/idsync-cron >> /etc/cron.d/idsync
-
-# If that failed, exit.
-rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
-
-# Remove original cron file without env vars
-rm -f /etc/cron.d/idsync-cron
-
-# If that failed, exit.
-rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
+# Dump env to a file to make available to cron
+env >> /etc/environment
```

Comment:
@fillup What did we decide about this change? Does it still work?

Reply:
it appears to be working in appsdev